projects/js-packages/components/components/icons/style.module.scss

```diff
@@ -0,0 +1,34 @@
+@import '@automattic/jetpack-base-styles/gutenberg-base-styles';
+
+.iconWrapper {
+	fill: #2C3338;
+}
+
+.star-icon {
+	fill: black;
+}
+
+.jetpack,
+.checkmark-icon {
+	fill: var(--jp-green-primary);
+}
+
+.socialIcon {
+	fill: $gray-700;
+	&.facebook {
+		fill: var( --color-facebook );
+	}
+	&.twitter {
+		fill: var( --color-twitter );
+	}
+	&.linkedin {
+		fill: var( --color-linkedin );
+	}
+	&.tumblr {
+		fill: var( --color-tumblr );
+	}
+	&.google {
+		fill: var( --color-gplus );
+	}
```

Comment:
Now, we don't have these `CSS Vars.` Suggest to hardcode it for now, and we can move to CSS Vars in some moment.

Reply:
There's a PostCSS script that substitutes these in Jetpack. It's a bit weird, and I didn't want to break something else.